activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -328,6 +328,8 @@ def safe_constantize(camel_cased_word)
         e.name.to_s == camel_cased_word.to_s)
     rescue ArgumentError => e
       raise unless /not missing constant #{const_regexp(camel_cased_word)}!$/.match?(e.message)
+    rescue LoadError => e
+      rause unless /Unable to autoload constant/.match?(e.message)
```

Comment:
rause?

Reply:
We might also want to add `#{const_regexp(camel_cased_word)}` to the end of this regex to ensure we're catching the right load error. Might be a good idea to test this actually raises now too seeing how CI is passing with a typo 🤔